.changelog/unreleased/bug/911-fix-header-validation-during-tmclient-update.md

```diff
@@ -0,0 +1,2 @@
+- Remove an incorrect validation during tendermint client update
```

Comment:
Should this be placed under the `.changelog/v0.44.2/bug-fixes` ?

Reply:
I wanted to do the changes on `release/v0.44.x` (based on `tags/v0.44.1`). You also have to create the other PR on `release/v0.44.x`. Then, perform a patch release for `release/v0.44.x` - whose current version is `0.44.1`.

Does it make sense?

Reply:
Oh, I see. Gotcha 👍🏻